### ISSUE TITLE
Empty placeholders no longer work in 6.0.0-Beta

### ISSUE THREAD

**Issue by rglidden** [flyway/flyway#2308]
###### Which version and edition of Flyway are you using?

Flyway Community Edition 6.0.0-beta

###### If this is not the latest version, can you reproduce the issue with the latest one as well?

Issue only occurs in 6.0.0-beta. Works as expected in 5.2.4

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)

Command-Line

###### Which database are you using (type & version)?

SQL Server 2017 Developer Edition

###### Which operating system are you using?

Windows 10

###### What did you do?

Use a placeholder in a script, then set the value of that placeholder to an empty string or white space only via the flyway.conf (to effectively set the placeholder to "nothing").

Script:

``` sql
PRINT 'Hello ${HELLO_PLANET}'
```

Config:
```
flyway.placeholders.HELLO_PLANET=
```

###### What did you expect to see?

No errors.

###### What did you see instead?

```
ERROR: Unable to parse statement in Operational\baseline\10.9.0\Framework\V0.0.0_0__Test_Placeholders.sql at line 1 col 1: String index out of range: 0
```

Setting the placeholder to a non-empty value such as `flyway.placeholders.HELLO_PLANET=World` works as expected.

**Comment by axelfontaine** [comment]
Good catch! This is indeed something we didn't include in our tests so far. Will be fixed in time for 6.0.0.

**Comment by shark-horse** [comment]
For what it's worth, if you don't provide the placeholder at all (not just leave it empty), `info` fails in 5.2.4 IME (not tested thoroughly or anything). I can provide more info if it's needed, but it's no issue that I have to provide placeholders. Placeholders not required for `info` with 5.0.7. (Found this during an upgrade.)
